### PR TITLE
NMS-15336: provide an option to disable kafka offset provider

### PR DESCRIPTION
**Pull request by cgorantla**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-15336

### CODE REVIEW

**Comment by RangerRick** [comment]
`release-30.x` is a dead repo, it's unsupported and doesn't auto-merge. is this a one-off for a customer or something?

**Review by RangerRick — CHANGES_REQUESTED** [review]
talked with @cgorantla out of band and he's going to re-target this to the right branch

once that's done re-request and I'll approve

**Review by RangerRick — APPROVED** [review]

File: docs/modules/reference/pages/configuration/tuning-kafka.adoc [hunk 1]
```diff
@@ -64,4 +64,16 @@ single-topic=false
 ----
 org.opennms.core.ipc.rpc.kafka.single-topic=false
 ----
-Create this file if it does not already exist.
\ No newline at end of file
+Create this file if it does not already exist.
+
+== Disable the Kafka offset provider
+
+{page-component-title} Core automatically monitors consumer offsets on Kafka to provide lag related metrics.
```
Comment (Bonrob2) [inline docs/modules/reference/pages/configuration/tuning-kafka.adoc:71]:
```suggestion
{page-component-title} Core automatically monitors consumer offsets on Kafka to provide lag-related metrics.
```

**Review by j-white — APPROVED** [review]

**Review by RangerRick — APPROVED** [review]

**Review by Bonrob2 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**